Sources/NIO/ServerSocket.swift

```diff
@@ -13,7 +13,7 @@
 //===----------------------------------------------------------------------===//
 
 /// A server socket that can accept new connections.
-final class ServerSocket: BaseSocket {
```

Comment:
Do we know how much losing the `final` keyword costs us here?

Reply:
Nope.... that said i think it should be fine as it’s internal so the compiler should be able to figure out that there are no subclasses (except in the test code ). /cc @weissi

Reply:
I _think_ it will figure that out. It's internal so it'll know that there are no subclasses (except when compiled for testing). Maybe put a comment there though `/* final but tests */` or something.

Reply:
Sure will do

Reply:
Given that I have a whitespace nit, I think we normally do this `/* final but tests */ class ServerSocket: BaseSocket`